src/pytorch_lightning/lite/wrappers.py

```diff
@@ -115,6 +117,26 @@ def _convert_float_tensor(t: Tensor) -> Tensor:
         output = apply_to_collection(output, function=_convert_float_tensor, dtype=Tensor)
         return output
 
+    @overload
+    def state_dict(self, *, destination: T_destination, prefix: str = ..., keep_vars: bool = ...) -> T_destination:
+        ...
+
+    @overload
+    def state_dict(self, *, prefix: str = ..., keep_vars: bool = ...) -> T_destination:
+        ...
+
+    def state_dict(
+        self, *args: Any, destination: Optional[T_destination] = None, prefix: str = "", keep_vars: bool = False
+    ) -> Optional[T_destination]:
+        return self._original_module.state_dict(
+            destination=destination,  # type: ignore[type-var]
+            prefix=prefix,
+            keep_vars=keep_vars,
+        )
+
+    def load_state_dict(self, state_dict: Mapping[str, Any], strict: bool = True) -> Any:
```

Comment:
Doesnt this always return None?

Reply:
See nn.Module implementation, which has no types (torch 1.12). I'm adding what mypy infers. Please edit directly if you have another solution.

Reply:
it returns a NamedTuple with missing keys